tests/integration/actions_trigger_test.go

```diff
@@ -263,8 +265,8 @@ func TestSkipCI(t *testing.T) {
 				},
 			},
 			Message:   fmt.Sprintf("%s add bar", setting.Actions.SkipWorkflowStrings[0]),
-			OldBranch: "main",
-			NewBranch: "main",
+			OldBranch: "master",
+			NewBranch: "master",
```

Comment:
Why these branch name changes?

Reply:
Because the test function to create a PR uses master as default branch, and it doesn't work well with main :laughing:

Reply:
We could also rename every occurrence of master in the tests to main in a later PR, would be appropriate as it's our default branch nowadays

Reply:
Yeah I agree, sounds like a good refactor to do to avoid such noisy changes in the future.